apps/mark-scan/backend/src/app.diagnostics.test.ts

```diff
@@ -227,3 +273,93 @@ test('failure saving the readiness report', async () => {
     }
   );
 });
+
+function mockDriverStatus(status: PaperHandlerStatus) {
+  mockOf(driver.getPaperHandlerStatus).mockResolvedValue(status);
+}
+
+describe('paper handler diagnostic', () => {
+  test('success', async () => {
+    const electionDefinition = (
+      await readElection(DIAGNOSTIC_ELECTION_PATH)
```

Comment:
Why do you load an election definition from file, and accommodate that in your build step, rather than just load an election from `libs/fixtures`?

Reply:
A few reasons:

1. This is a user-facing mock election so we wanted the contests to make it abundantly clear it's a mock election with names like "Candidate A" and "Choice B". I believe our existing fixtures all use real-sounding names.

2. I was concerned that fixtures are associated with testing and don't have a stable contract. We'd be in danger of changes to fixtures accidentally breaking this production flow. If that's wrong or if we have a construct for mock elections used in production, I'd be happy to fold this into that construct!

Reply:
I think reason #1 makes sense and is a good enough reason. I think our testing should guard against #2. We don't have another construct but if we need an election definition for another ballot, would be excellent to use the one you've created here.